openfisca_aotearoa/variables/acts/social_security/disability_allowance/disability_allowance.py

```diff
@@ -0,0 +1,304 @@
+#     value_type = bool
+#     default_value = False
+#     entity = entities.Person
+#     definition_period = periods.WEEK
+#     label = "Is the person receiving an entitlement under Part 4 of the Veterans’ Support Act 2014?"
+#     reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#DLM6783280"
+
+
+class disability_allowance__income_limit_clause_10(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Does the person earn below the Disability Allowance Income Limit as per clause 10?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784890.html"
+
+    def formula_2018_11_26(persons, period, parameters):
+        person_aged_16_or_17 = (persons("age", period.start) >= 16) * (persons("age", period.start) <= 17) 
+        no_partners = numpy.logical_not(persons("person_has_partner", period))
+        income_within_limit = persons("social_security__income", period) <= parameters(period).disability_allowance.income_limits.clauses["clause_10"]
+        without_dependant_child = persons("social_security__dependent_children", period) == 0
+        return person_aged_16_or_17 * no_partners * income_within_limit * without_dependant_child 
+
+class disability_allowance__income_limit_clause_11(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Does the person earn below the Disability Allowance Income Limit as per clause 11?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784890.html"
+
+    def formula_2018_11_26(persons, period, parameters):
+        person_not_aged_16_or_17 = (persons("age", period.start) < 16) + (persons("age", period.start) > 17)
+        no_partners = numpy.logical_not(persons("person_has_partner", period))
+        without_dependant_child = persons("social_security__dependent_children", period) == 0
+        not_a_child = numpy.logical_not(persons.has_role(entities.Family.CHILD))  #review this
+        income_within_limit = persons("social_security__income", period) <= parameters(period).disability_allowance.income_limits.clauses["clause_11"]
+        return person_not_aged_16_or_17 * no_partners * without_dependant_child * income_within_limit * not_a_child
+
+
+class disability_allowance__income_limit_clause_12(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Does the person earn below the Disability Allowance Income Limit as per clause 12?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784890.html"
+
+    def formula_2018_11_26(persons, period, parameters):
+        in_relationship = persons("person_has_partner", period) #refactor this to person_has_partner
```

Comment:
If you rebase from main you'll see that I rationalised the usde of this varfiabvle